site/content/data-platform/features.md

```diff
@@ -18,57 +18,82 @@ scaling, and management. Powered by the official
   Arango Data Platform for centralizing authentication and routing.
 
-- **Web interface**: The Platform includes a unified, browser-based UI that lets
-  you access its features in an intuitive way. Optional products like the
-  AI Suite seamlessly integrate into the UI if installed.
+## Platform Components
 
-## Kubernetes Integration
+The Arango Data Platform consists of three integrated layers that work together
+to provide a complete, enterprise-ready solution:
 
-At its core, the Arango Data Platform is purpose-built for Kubernetes environments, leveraging the 
-[official ArangoDB Kubernetes Operator](https://arangodb.github.io/kube-arangodb/docs/) 
-(`kube-arangodb`) to deliver enterprise-grade automation, scalability, and operational excellence.
+### Layer 1: ArangoDB Enterprise Edition (Foundation)
 
-## Features
+At the foundation is [**ArangoDB Enterprise Edition**](../../arangodb/3.12/), the 
+powerful multi-model database that provides:
 
-The Arango Data Platform provides these core capabilities out of the box:
+- **Graph**: Native graph database capabilities with efficient traversals and pattern matching
+- **Document**: Flexible JSON document storage with schema validation
+- **Key-Value**: High-performance key-value operations
+- **Vector**: Vector embeddings and similarity search for AI applications
+- **Search**: Full-text search and complex query capabilities
 
-- [**ArangoDB Core**](../arangodb/3.12/_index.md): The ArangoDB database system with support for
-  graphs, documents, key-value, full-text search, and vector search.
+### Layer 2: Graph Visualizer
 
-- [**Graph Visualizer**](graph-visualizer.md):
-  A web-based tool for exploring your graph data with an intuitive interface and
-  sophisticated querying capabilities.
+The [**Arango Visualizer**](graph-visualizer/) provides an intuitive web-based interface
+that brings your data to life with:
 
-## Extend the Arango Data Platform with AI capabilities
+- **Graph Exploration**: Interactive visualization of graph structures with
+  drag-and-drop navigation, zoom controls, and customizable node appearances
+  
+- **Smart Search**: Intelligent search capabilities to quickly find nodes, edges,
+  and patterns within your graphs using natural language queries
+  
+- **Visual Layouts**: Multiple automatic layout algorithms (force-directed, hierarchical,
+  circular) to best represent your data relationships
+  
+- **Team Workspaces**: Collaborative features that allow teams to share visualizations,
+  queries, and insights across the organization
 
-Take your Arango Data Platform to the next level with the [**AI Suite**](../ai-suite/_index.md) that offers advanced AI and machine learning capabilities that integrate seamlessly into the platform's unified web interface.
+The Graph Visualizer seamlessly integrates with the database layer and provides the
+primary interface for data exploration and analysis.
 
-What you get with the AI Suite:
+### Layer 3: Arango Platform Suite
```

Comment:
This makes more sense as layer 2 in the bigger picture, but the visualizer doesn't really use any of these features directly I think?